sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -101,8 +101,9 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
   def unpivotValDataTypeMismatchError(values: Seq[NamedExpression]): Throwable = {
```

Comment:
uhoh, seems like it fails in Scala 2.12 now 😢 https://github.com/G-Research/spark/runs/7601695538?check_suite_focus=true